src/NodaTime.Test/NodaConstantsTest.cs

```diff
@@ -17,5 +18,21 @@ public void JulianEpoch()
             var epoch = localEpoch.InZoneStrictly(DateTimeZone.Utc).ToInstant();
             Assert.AreEqual(epoch, NodaConstants.JulianEpoch);
         }
+
+        [Test]
+        public void BclTicksAtEpoch()
+        {
+            Assert.AreEqual(
+                new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc).Ticks,
+                NodaConstants.BclTicksAtUnixEpoch);
+        }
+
+        [Test]
+        public void BclDaysAtEpoch()
+        {
+            Assert.AreEqual(
+                (int) (new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc).Ticks / NodaConstants.TicksPerDay),
+                NodaConstants.BclDaysAtUnixEpoch);
```

Comment:
Could we test by multiplying rather than a truncating division? I know it's not testing directly, but I suspect this would still pass if `DateTime.Ticks` weren't exactly divisible by `TicksPerDay`.